content/watch/competitions/cccc.md

```diff
@@ -12,6 +12,15 @@ Organizers pick participants based on engine's entertainment value. Particularly
 
 | Season                  | Year | Time control   | Lc0 final position   |
 | ----------------------- | ---- | -------------- | -------------------- |
+| CCC 24: Bullet          | 2025 | 1+1            | 4                    |
```

Comment:
That's from another change (you need to merge master back into your branch, or rebase)